### PR TITLE
Allow setting pyodide version

### PR DESCRIPTION
**Pull request by ahuang11**
Closes https://github.com/pawamoy/markdown-exec/issues/66

I probably am missing a few pieces.

### CODE REVIEW

**Comment by pawamoy** [comment]
Awesome, thank you for the super fast PR!

Looking at the existing code, I think `version` will end up in the `extra` dict. Try to pop it from there. Other than that, can you use `.format` instead of `%`?

**Comment by pawamoy** [comment]
Don't pay attention to CI failures, they're unrelated.

**Comment by ahuang11** [comment]
> it was easier to use % in the template variable because there are curly braces in there 😄

Aha yeah, I was wondering why the old school % was used over format, but that's a good point!

**Review by pawamoy — APPROVED** [review]
Looks perfect, thanks! Can you just confirm it actually works (since Pyodide was upgraded), and whether it fixes #66? :smile:

File: src/markdown_exec/formatters/pyodide.py [hunk 1]
```diff
@@ -46,6 +46,7 @@
 def _format_pyodide(code: str, md: Markdown, session: str, extra: dict, **options: Any) -> str:  # noqa: ARG001
     global _counter  # noqa: PLW0603
     _counter += 1
+    version = extra.get("version", "0.26.4").lstrip("v")
```
Comment (ahuang11) [inline src/markdown_exec/formatters/pyodide.py:49]:
```suggestion
    version = extra.pop("version", "0.26.4").lstrip("v")
```

**Comment by pawamoy** [comment]
Oh we'll need some docs, though I can take care of that if you're unsure about it.

**Comment by ahuang11** [comment]
Yeah if you can handle docs that'd be great!

**Comment by pawamoy** [comment]
OK looks like it's working well, and the version upgrade also fixes #66. Thank you so much!

**Comment by ahuang11** [comment]
Thanks! It seems to be working past the installation.

However, I don't think it's rendering the Panel widgets. Might have to research a bit on https://github.com/holoviz-dev/nbsite/tree/main/nbsite/pyodide and see what's done differently.

image

**Pull request merged**